### ISSUE TITLE
Possibility to cancel the editing of the language of an existing sentence

### ISSUE THREAD

**Issue by Guybrush88** [Tatoeba/tatoeba2#2605]
**Story** or **Context**
When browsing sentences, especially from a mobile device, one might tap/click by accident the flag of the sentence instead of the translation button, so the dropdown menu with all the languages appear, and it won't go away unless the user refreshes the page or changes the language.

**Idea**
As an improvement for the behavior described above, there could be a 'cancel' button that allows users to close the dropdown menu without refreshing the page or applying any change.

**Comment by AndiPersti** [comment]
I guess you are talking about the old design, aren't you?

Clicking/Tapping on the flag again closes the language drop-down.

**Comment by Guybrush88** [comment]
> I guess you are talking about the old design, aren't you?
> 
> Clicking/Tapping on the flag again closes the language drop-down.

yes, I mean the old design, sorry if I didn't specify. Thanks for the tip
